changelog/147.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:improvement
-op: rename seeker to Op.
-`
+op: Rename seeker to Op.
```

Comment:
hehe, oopsie. Thanks 🙏

Reply:
Haha, I TOTALLY missed this in the other PRs!! I decided to see what our changelogs looked like, and there were some missing, which is the only reason I noticed this. 😄